src/locales/en/common.json

```diff
@@ -574,12 +574,11 @@
     },
     "import": {
       "scan": {
-        "title": "Scan QR Code",
+        "title": "Scan LiveQR code",
         "descTop": {
-          "line1": "Please open Ledger Live desktop and go to",
-          "line2": "Settings > Mobile > Export"
+          "line1": "In Ledger Live desktop, go to:",
+          "line2": "Settings > Export accounts > Export"
         },
-        "descBottom": "Please put the QR code within the square"
```

Comment:
this is a removal of wording. it affects the design. we need a second eye from design and then we need a dev to remove the location if approved.

Reply:
Design is already OK : https://zpl.io/2jw9wqO